lib/globals.ts

```diff
@@ -1,3 +1,7 @@
 export const HOST = process.env.HUBOT_HOST
-export const SECOND = 1000
+export const MILLISECOND = 1000
+export const SECOND = 1
```

Comment:
This should be `1 * MILLISECOND`—if we need to do a division later on by `MILLISECOND` to have the right precision for e.g. the Discord API, let's do that there.

We can probably drop a comment on these indicating that they're in millisecond precision.

The reasoning to do that rather than reindex everything against seconds is:
- More precision is good in many cases, until it's time to drop to lower precision.
- `30 * MILLISECOND` (as used below) isn't correct—we're not setting an interval for 30 milliseconds, but rather for 30 seconds. The goal of the constants here is so that the phrase that uses it reflects the intent, and to do that across most use cases successfully, we need to use the higher precision as the basis.

Reply:
Ok that makes sense! So then for the Discord API, since we need exact seconds for DAY, WEEK instances we will have to do `WEEK / MILLISECOND` in-order to get the desired `maxAge: 604800` vs `604800000` by default. Unless you'd prefer we just add that into the `global.ts` scope, something along the lines of `WEEK_IN_SECONDS`.